k6-stress-test.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable max-len */
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+// Scale the number of request per second: 1, 10, 100, 1K
+export const options = {
+  stages: [
+    // { duration: '1s', target: 1 }, // target is number of virtual users, duration is how many per minute
+    // { duration: '5m', target: 1 },
+    // { duration: '2m', target: 10 },
+    { duration: '5m', target: 10 },
+    { duration: '2m', target: 100 },
+    { duration: '5m', target: 100 },
+    { duration: '2m', target: 1000 },
+    { duration: '5m', target: 1000 },
+  ],
+};
+export default function () {
+  const url = 'http://localhost:3003/questions';
+  let headers = { 'Content-Type': 'application/x-www-form-urlencoded' };
+  let data = {
+    product_id: 7,
+    question_text: 'Stress Test 10:00AM Oct 26 2020',
+    question_date: '2020-10-22',
+    question_user_email: 'karebae@gmail.com',
+    question_username: 'karebae',
+    question_user_location: 'San Francisco, CA',
+  };
+  let res = http.post(url, data, { headers: headers });
+  headers = { 'Content-Type': 'application/x-www-form-urlencoded' };
+  res = http.post(url, data, { headers: headers });
+
```

Comment:
Awesome to see you have K6 running. I may have questions when I implement my tests.